docs/content/concepts/assets/asset-checks/checking-for-data-freshness.mdx

```diff
@@ -0,0 +1,339 @@
+---
+title: "Using asset checks to check data freshness | Dagster Docs"
+description: "Use freshness checks to identify data assets that are overdue for an update."
+---
+
+# Using asset checks to check data freshness
+
+A freshness check is a type of [asset check](/concepts/assets/asset-checks) that allows you to define the amount of time between materializations/observations for Dagster [assets](/concepts/assets/software-defined-assets).
+
+Using freshness checks, you can easily identify assets that are overdue for an update. As freshness checks don't depend on a specific root cause, it makes them helpful in accounting for unknowns. For example, freshness checks can identify overdue assets caused by any of the following:
+
+- The pipeline hitting an error and failing
+- Runs were never scheduled
+- A backed up run queue
+- Runs are taking longer than expected to complete
+
+By the end of this guide, you'll understand what freshness checks are and how to implement them in your data pipelines.
+
+---
+
+## How it works
+
+An asset is considered fresh if it's been materialized or observed within a defined time window. Otherwise, assets are considered overdue and in need of an update. To identify these assets, you can use **freshness checks**.
+
+Let's say an asset begins materializing at 12:00AM and takes 10 minutes to complete. Allowing for operational constraints and delays, the asset should always be materialized by 12:30AM. Using a freshness check, we can define the time that the asset should be updated. If it's not, Dagster will mark it as **Overdue**.
```

Comment:
If the goal is provide a motivating example, I think a realistic one would be to talk about a situation where data is expected to be updated by 7 am every day, so that people reviewing dashboards will have access to up-to-date data.

Reply:
Good call!